tools/identity-resolution/Helpers/GitHubToAADConverter.cs

```diff
@@ -77,7 +77,7 @@ private async Task<UserLink[]> GetPeopleLinksAsync()
             try
             {
                 // This is aad scope of opensource rest API.
-                string[] scopes = new [] { "api://66b6ea26-954d-4b68-8f48-71e3faec7ad1/.default" };
+                string[] scopes = new [] { "66b6ea26-954d-4b68-8f48-71e3faec7ad1/.default" };
```

Comment:
When testing I actually didn't get the token to work with using the scope I had to do a resource id of "api://66b6ea26-954d-4b68-8f48-71e3faec7ad1"

Reply:
I verified with local testing using SP login to az cli